code/modules/events/wizard/magicarp.dm

```diff
@@ -50,7 +53,9 @@
 	color = "#00FFFF"
 	maxHealth = 75
 	health = 75
+	chaos = TRUE
 
 /mob/living/simple_animal/hostile/carp/ranged/chaos/Shoot()
-	projectiletype = pick(allowed_projectile_types)
+	if(chaos)
+		projectiletype = pick(allowed_projectile_types)
```

Comment:
If you're going to make chaos a var this check needs to be in the type that adds that var

Reply:
why does the chaos carp need to check for chaos lol